README.md

```diff
@@ -1,6 +1,6 @@
 # GEFF Terraform Module
 
-The Generic External Function Framework (GEFF) is extensible Python code which can be called using a [Snowflake External Function](https://docs.snowflake.com/en/sql-reference/external-functions-introduction.html) that allows Snowflake operators to perform generic invocations of Call Drivers (e.g. HTTP, SMTP, XML-RPC) and either return or write responses to generic Destination Drivers (e.g. S3). This empowers them to create new pipelines in Data Infrastructure while using reviewed and standardized RBAC and interaction with Cloud Infrastructure for secrets management.
+The Generic External Function Framework (GEFF) is extensible Python Lambda code which can be called using a [Snowflake External Function](https://docs.snowflake.com/en/sql-reference/external-functions-introduction.html) that allows Snowflake operators to perform generic invocations of Call Drivers (e.g. HTTP, SMTP, XML-RPC) and either return or write responses to generic Destination Drivers (e.g. S3). This empowers them to create new pipelines in Data Infrastructure while using reviewed and standardized RBAC and interaction with Cloud Infrastructure for secrets management.
```

Comment:
I intentionally skipped the hosting framework here -- it happened that our first GEFF was implemented via, but it can be hosted on ECS or k8s or Serverless just as well as a "pure" Lambda. it could also be central service hosted by Snowflake, or a third party.

Reply:
We'll have to make a different one maybe? Cuz this one is only meant to work with Lambda. That's why it has the `lambda_function.py`.

We can make a roadmap item to make a pypi library that we can import into any platform and essentially import the GEFF functionality?

Reply:
@sfc-gh-afedorov Also maybe we should move this discussion to the [geff](https://github.com/Snowflake-Labs/geff/) repo. Because this README.md will change to just refer to GEFF's documentation and introduce the terraform that it will create in order to power GEFF.